qiskit/terra/quantum_circuits.ipynb

```diff
@@ -149,7 +149,7 @@
    "source": [
     "## Quantum Circuits\n",
     "\n",
-    "Quantum Circuits are made using registers. Either when initiated or by using the `add_register` command. "
+    "Quantum Circuits are made using registers, which are created either when initiated or by using the `add_register` command. "
```

Comment:
Quantum Circuits are made using registers.  These registers may be specified during initialization, or added afterwards using the `add_register` method.